Dungeoneer/src/com/interrupt/dungeoneer/dto/LookAtDTO.java

```diff
@@ -0,0 +1,27 @@
+package com.interrupt.dungeoneer.dto;
+
+import com.badlogic.gdx.graphics.Color;
+
+public class LookAtDTO {
```

Comment:
Should rename this to `LookAtInfo` 
1. I like that name more (I had to look up what a DTO was)
2. All the methods to get the object call it a LookAtInfo 😄

Reply:
Fixed.